SU-SWS/saa_victoria_subtheme
# Adapt 1194  paragraph behaviors base

# READY FOR REVIEW

# Summary
- Adding basic ground work for paragraph variants - note it is not finished yet, will continue in many other pr's and branches to come.

# Review By (Date)
- 2/11 or asap just to continue working.

# Urgency
- low
# Steps to Test

1. Just look at code. If you want to confirm the basic functionality, you can pull down a branch, clear cache, and make sure when you edit a card you see a STYLE tab with 2 basic fields (card orientation and border style) But I will be continuing work on this and it's not ready for actual use. It does not actually do anything besides add those fields right now.


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-1194
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
